### DISCUSSION TITLE
Presidio (data anonymisation)

### DISCUSSION THREAD

**Discussion by bubonicfred** [jonaswinkler/paperless-ng#389] [Feature Requests]
Thought I'd chuck this here for future reference.
https://github.com/microsoft/presidio
Microsofts open source tool for anonymising sensitive info in text and images (SSN, Credit card etc.)
This could possibly help with not being able to encrypt text without it being searchable.
V2 will be python based and pip installable so worth waiting for?

**Comment by jonaswinkler** [comment]
I'm not exactly sure what we should use this for? What's the use case?
